### PR TITLE
Fix torch_vision_tutorial.rst

### LINKED ISSUE #960: Mask-RCNN tutorial one-line change suggestion
I would like to suggest changing the labels in [Mask-RCNN tutorial](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html) from 

`labels = torch.ones((num_objs,), dtype=torch.int64)`
 
to the following:

`label = torch.as_tensor(obj_ids, dtype=torch.int64)`

that will work if there are more than 2 types of object instances (`num_objs`) in the image. In fact, the new form will give the same exact vector of `labels = torch.ones((num_objs,), dtype=torch.int64)` and will work for any number of object instances. I have tried it with a dataset containing 60 object instances and all went well. 

Kind regards

### PR DESCRIPTION
**Pull request by ver2king**
Fixes #960

## Checklist

- [x] The issue that is being fixed is referred to in the description (see above "Fixes #960")
- [x] Fix the code line: 

**labels = torch.ones((num_objs,), dtype=torch.int64)**

to the following:

**label = torch.as_tensor(obj_ids, dtype=torch.int64)**

cc @datumbox @nairbv @fmassa @NicolasHug @YosuaMichael

### CODE REVIEW

**Comment by svekars** [comment]
I'm not sure why we are modifying anything all these other files apart from ``intermediate_source/torchvision_tutorial.rst``

**Review by kit1980 — CHANGES_REQUESTED** [review]
@ver2king your PR currently includes 16 files, including your ".idea" folder.

On the tutorial change itself, you need to update usages of the variable after the rename.
Also please verify the changes by running the code, don't submit blindly.
